frontend_lib/src/helper.js

```diff
@@ -799,3 +799,27 @@ export const getCoverBaseUrl = (apiUrl, userId) => `${apiUrl}/users/${userId}/co
 export const getFileDownloadUrl = (apiUrl, workspaceId, contentId, filename) => `${apiUrl}/workspaces/${workspaceId}/files/${contentId}/raw/${filename}?force_download=1`
 
 export const htmlToText = (domParser, htmlString) => domParser.parseFromString(htmlString, 'text/html').documentElement.textContent
+
+export const addExternalLinksIcons = (htmlString) => {
+  const doc = getDocumentFromHTMLString(htmlString)
+  const locationUrl = new URL(location.toString())
```

Comment:
If you get a linting error here, you may be able to fix it like this
```suggestion
  const locationUrl = new URL(window.location.toString())
```

Reply:
fixed